lib/db/db-client.ts

```diff
@@ -59,6 +59,52 @@ const initializer = combine(databaseSchema.parse({}), (set, get) => ({
     )
   },
 
+  getFileByPath: (file_path: string) => {
+    const state = get()
+    return state.files.find((f) => f.file_path === file_path)
+  },
```

Comment:
It would be better if we can normalize the file_path before doing any operation on the path

Reply:
For other operations as well